Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -385,6 +385,31 @@ void ACesium3DTileset::SetTilesetSource(ETilesetSource InSource) {
   }
 }
 
+bool CompareMaps(
```

Comment:
This should go in the anonymous namespace.

Reply:
Also something like `MapsAreEqual` would be a better name than `CompareMaps`, because the latter doesn't indicate as clearly what it means when the function returns true or false.